plugins/services/src/js/utils/DeclinedOffersUtil.js

```diff
@@ -0,0 +1,79 @@
+import DeclinedOffersReasons from '../constants/DeclinedOffersReasons';
+
+const DecinedOffersUtil = {
+  getSummaryFromQueue(queue) {
+    const {app, lastUnusedOffers, processedOffersSummary} = queue;
+
+    if (processedOffersSummary == null
+      || processedOffersSummary.unusedOffersCount === 0
+      || lastUnusedOffers == null
+      || lastUnusedOffers.length === 0) {
+      return null;
+    }
+
+    const {processedOffersCount = 0, rejectReason = {}} = processedOffersSummary;
+    const matchedRoleOffers = processedOffersCount -
+      (rejectReason[DeclinedOffersReasons.UNFULFILLED_ROLE] || 0);
+    const matchedConstraintOffers = matchedRoleOffers -
+      (rejectReason[DeclinedOffersReasons.UNFULFILLED_CONSTRAINT] || 0);
+    const matchedCPUOffers = matchedConstraintOffers -
+      (rejectReason[DeclinedOffersReasons.INSUFFICIENT_CPU] || 0);
+    const matchedMemOfffers = matchedConstraintOffers -
```

Comment:
Hahaha, good catch... this awesome variable is no longer needed after the API changes :)